pkg/plugins/resources/updateclihttp/main.go

```diff
@@ -0,0 +1,89 @@
+package updateclihttp
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/sirupsen/logrus"
+	"github.com/updatecli/updatecli/pkg/core/httpclient"
+)
+
+// Http defines a resource of type "http"
+type Http struct {
+	spec       Spec
+	httpClient httpclient.HTTPClient
+	httpReq    *http.Request
+}
+
+// New returns a reference to a newly initialized (TODO)
```

Comment:
Is the "TODO" a left over?

Reply:
oh I forgot to fill it. Let me update, thanks for the catch!

Reply:
Treated in https://github.com/updatecli/updatecli/pull/1773/commits/494ac44c90ba03bc98496c37b724efa2a894ff8c